static/app/views/performance/queues/destinationSummary/transactionsTable.tsx

```diff
@@ -83,38 +81,14 @@ interface Props {
   domain?: string;
   error?: Error | null;
   meta?: EventsMetaType;
-  pageLinks?: string;
 }
 
-export function TransactionsTable({error, pageLinks}: Props) {
+export function TransactionsTable({error}: Props) {
   const organization = useOrganization();
   const location = useLocation();
   const destination = decodeScalar(location.query.destination);
-  const cursor = decodeScalar(location.query?.[QueryParameterNames.DOMAINS_CURSOR]);
 
-  const mutableSearch = new MutableSearch(DEFAULT_QUERY_FILTER);
-  // TODO: This should filter by destination, not transaction.
-  // We are using transaction for now as a proxy to demo some functionality until destination becomes a filterable tag.
-  if (destination) {
-    mutableSearch.addFilterValue('transaction', destination);
-  }
-  const {data, isLoading, meta} = useSpanMetrics({
-    search: mutableSearch,
-    fields: [
-      'transaction',
-      'count()',
-      'count_op(queue.submit.celery)',
-      'count_op(queue.task.celery)',
-      'sum(span.self_time)',
-      'avg(span.self_time)',
-      'avg_if(span.self_time,span.op,queue.submit.celery)',
-      'avg_if(span.self_time,span.op,queue.task.celery)',
-    ],
-    sorts: [],
-    limit: 10,
-    cursor,
-    referrer: 'api.starfish.http-module-landing-domains-list',
-  });
+  const {data, isLoading, meta, pageLinks} = useQueuesByTransactionQuery({destination});
```

Comment:
Should the `error` from this query make it down to the `error` prop of the `GridEditable`?

Reply:
Good point, done!

Reply:
Nice!